build-system/tasks/visual-diff/index.js

```diff
@@ -748,13 +748,14 @@ async function ensureOrBuildAmpRuntimeInTestMode_() {
       log(
         'fatal',
         'The AMP runtime was not built in test mode. Run',
-        colors.cyan('gulp build|dist --fortesting'),
+        colors.cyan('gulp dist --fortesting'),
         'or remove the',
         colors.cyan('--nobuild'),
         'option from this command'
       );
     }
   } else {
+    argv.compiled = true;
```

Comment:
`argv` is a module-level variable, so this can break if `buildRuntime` ends up using its own module-level copy of `argv`.

A more resilient way to implement this is to add a boolean argument called `compiled` to `buildRuntime()` and update calling sites to pass in `argv.compiled`, `true`, or `false` as the case may be.

Reply:
👍